mlfinlab/data_structures/data_structures.py

```diff
@@ -106,13 +117,16 @@ def _assert_dataframe(test_batch):
     :param test_batch: DataFrame which will be tested.
     """
     assert test_batch.shape[1] == 3, 'Must have only 3 columns in csv: date_time, price, & volume.'
-    assert isinstance(test_batch.iloc[0, 1], float), 'price column in csv not float.'
-    assert isinstance(test_batch.iloc[0, 2], np.int64), 'volume column in csv not int.'
+    assert isinstance(test_batch.iloc[0, 1],
```

Comment:
IMO, it's a bad idea to use indexes. A little change can lead to huge bugs. Is it possible to change index references to name references ?